From 4524f5d6b7d90684f2b205e472cd65a682d5fab5 Mon Sep 17 00:00:00 2001
From: Goldwyn Rodrigues <rgoldwyn@suse.de>
Date: Wed, 29 Nov 2023 11:36:23 -0600
Subject: [PATCH] tests: increase delay for waitpid test
References: bsc#1217651

In some test executions on s390x, the waitpid test fails because 1
gets printed before 2.

[  557s] --- /home/abuild/rpmbuild/BUILD/util-linux-2.39.2/tests/expected/misc/waitpid-normal	2023-06-14 09:11:15.910887765 +0000
[  557s] +++ /home/abuild/rpmbuild/BUILD/util-linux-2.39.2/tests/output/misc/waitpid-normal	2023-11-27 23:30:30.406675022 +0000
[  557s] @@ -1,4 +1,4 @@
[  557s]  3
[  557s] -2
[  557s]  1
[  557s] +2
[  557s]  4

Increase the time to print 1, so the test numbers are printed in
expected order.

Signed-off-by: Goldwyn Rodrigues <rgoldwyn@suse.com>

diff --git a/tests/ts/misc/waitpid b/tests/ts/misc/waitpid
index daed74fe0..904222c05 100755
--- a/tests/ts/misc/waitpid
+++ b/tests/ts/misc/waitpid
@@ -24,7 +24,7 @@ ts_check_test_command "$TS_CMD_WAITPID"
 
 ts_init_subtest normal
 
-(sleep 0.2; echo 1 >> "$TS_OUTPUT") &
+(sleep 0.5; echo 1 >> "$TS_OUTPUT") &
 BG1="$!"
 
 (sleep 0.1; echo 2 >> "$TS_OUTPUT") &
